packages/api/amplify_api/example/lib/graphql_api_view.dart

```diff
@@ -62,127 +69,549 @@ class _GraphQLApiViewState extends State<GraphQLApiView> {
+                    ),
+                    child: const Text('Blog'),
+                  ),
+                ),
+                Flexible(
+                  child: ElevatedButton(
+                    onPressed: widget.isAmplifyConfigured && _blog != null
+                        ? createPost
+                        : null,
+                    style: ElevatedButton.styleFrom(
+                      backgroundColor: Colors.amber, // This is what you need!
+                    ),
+                    child: const Text('Post'),
+                  ),
+                ),
+                Flexible(
+                  child: ElevatedButton(
+                    onPressed: widget.isAmplifyConfigured &&
+                            _blog != null &&
+                            _post != null
+                        ? createComment
+                        : null,
+                    style: ElevatedButton.styleFrom(
+                      backgroundColor: Colors.amber, // This is what you need!
+                    ),
+                    child: const Text('Comment'),
+                  ),
+                ),
+              ],
+            ),
+
+            /// Subscriptions
+            ///
+            const SizedBox(
+              width: double.infinity,
+              child: Text(
+                'Subscriptions',
+                textAlign: TextAlign.left,
+              ),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                ElevatedButton(
+                  onPressed: widget.isAmplifyConfigured && _subscription == null
+                      ? subscribe
+                      : null,
+                  child: const Text('Blogs'),
+                ),
+                ElevatedButton(
+                  onPressed: widget.isAmplifyConfigured &&
+                          _subscriptionByID == null &&
+                          _blog != null
+                      ? subscribeByID
+                      : null,
+                  child: const Text(
+                    'Posts By BlogID',
+                    maxLines: 1,
+                  ),
+                ),
+                ElevatedButton(
+                  onPressed: _subscription != null
+                      ? () => setState(() {
+                            _unsubscribe?.call();
+                            _unsubscribe = null;
+                            _subscription = null;
+                          })
+                      : null,
+                  child: const Text('Unsubscribe'),
+                ),
+              ],
+            ),
+            const Padding(padding: EdgeInsets.all(10)),
+
+            const SizedBox(
+              width: double.infinity,
+              child: Text(
+                'Authorization Type',
+                textAlign: TextAlign.left,
+              ),
+            ),
+            Row(
```

Comment:
This row overflows for me. You can use a `Wrap` instead or a `ListView` with `Axis.horizontal` to have a scrolling Row